modules/ocf_mirrors/templates/record-mirrors-stats.py.erb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+import os
+import pymysql
+from datetime import date
+
+OCFSTATS_PWD = "<%= @ocfstats_password %>"
+MIRRORS_PATH = '/opt/mirrors/ftp'
+LOG_PATH = '/var/log/apache2'
+LOG_NAME = 'mirrors.ocf.berkeley.edu_access.log'
+
+sources = [mirrored for mirrored in os.listdir(MIRRORS_PATH)
+           if os.path.isdir(os.path.join(MIRRORS_PATH, mirrored)) and not mirrored.startswith('.')]
+
+sources.append('other') # catchall
+
+def build_dists():
+    return { mirrored: { 'up': 0, 'down': 0 } for mirrored in sources }
+
+def process_file(fn):
```

Comment:
2 blank lines before/after lines per PEP8 (going to stop commenting these, pre-commit should do this for you)